### PR TITLE
Release Notes 4.4.a

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds the release notes for 4.4.a

## Changed Pages

💻 [Preview URL for Page](https://deploy-preview-3320--docs-spectrocloud.netlify.app/release-notes/)

## Jira Tickets

🎫 [DOC-1269](https://spectrocloud.atlassian.net/browse/DOC-1269)
🎫 [PLT-1298](https://spectrocloud.atlassian.net/browse/PLT-1298)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._
Release PR

[DOC-1269]: https://spectrocloud.atlassian.net/browse/DOC-1269?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PLT-1298]: https://spectrocloud.atlassian.net/browse/PLT-1298?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/release-notes/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,177 @@ tags: ["release-notes"]
...
+  outbound traffic. The proxy configuration is set in the **values.yaml** file during the deployment process. Refer to
+  the
+  [Self-Hosted Helm Chart Configuration Reference](../enterprise-version/install-palette/install-on-kubernetes/palette-helm-ref.md#reach-system)
+  or the [Deploy a PCG to an Existing Kubernetes Cluster](../clusters/pcg/deploy-pcg-k8s.md) guide to learn more.
+
+- Improvements to the Palette agent has reduced the frequency and bandwidth of agent communication with the Palette
+  management platform. This change reduces the resource consumption by the Palette agent in a cluster and the bandwidth
+  usage between the agent and the Palette management platform.
+
+- Palette API responses now include the header`Cache-Control`. This header provides information on how long the response
```
Comment (lennessyy) [inline docs/docs-content/release-notes/release-notes.md:52]:
```suggestion
- Palette API responses now include the header `Cache-Control`. This header provides information on how long the response
```

File: docs/docs-content/release-notes/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,178 @@ tags: ["release-notes"]
...
+  more.
+
+#### Improvements
+
+- You can now disable password changes of Operating System (OS) users from Local UI. When password updates from Local UI
+  are disabled, you can still update the OS user password from the OS or the Edge Management API. Check out the
+  [Access Local UI](../clusters/edge/local-ui/host-management/access-console.md) page to learn more.
+
+- Several enhancements have been made to the UI for Edge host management in the context of cluster creation and updates.
+  The new changes ensure a consistent and user-friendly experience that involves new designs for the edge host selection
```
Comment (lennessyy) [inline docs/docs-content/release-notes/release-notes.md:92]:
```suggestion
  The new changes ensure a consistent and user-friendly experience that involves new designs for the Edge host selection
```

Comment (lennessyy) [reply docs/docs-content/release-notes/release-notes.md:92]: This sentence reads slightly awkward. Maybe:

> These changes ensure a consistent and user-friendly experience, including new designs for the Edge host selection screen and a customizable, powerful grid view. This redesign provides a better user experience for managing many edge hosts in large-scale environments. Check out the [Edge Host Grid View](../clusters/edge/site-deployment/edge-host-view.md) page to learn more.

**Review by lennessyy — APPROVED** [review]
Looks great - I rewrote parts of the Edge section to put some of the changes in more context.

File: docs/docs-content/release-notes/release-notes.md (hunk 2 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/release-notes/release-notes.md:92]: Thank you Lenny

**Pull request merged**